### ISSUE TITLE
Set minSdkVersion to 24 (Android 7 "Nougat")

### ISSUE THREAD

**Issue by mahibi** [nextcloud/talk-android#2886]
Android 6 was used by 0,44% so i will be dropped

**Comment by AndyScherzinger** [comment]
cc @tobiasKaminsky

**Comment by tobiasKaminsky** [comment]
I remember that we agreed in some Android meeting to always be in sync between (nowadays) files, talk, library and notes.
So next time please ask, so that we can check on other apps if this also fits :)

**Comment by tobiasKaminsky** [comment]
I now saw that Android 6.0 is on Files also <0,5%, so this is fine.

**Comment by tobiasKaminsky** [comment]
Same for Notes.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky good point, to have a little reminder I updated the major release issue template to reflect this at https://github.com/nextcloud/talk-android/issues/2884

**Comment by mahibi** [comment]
> I remember that we agreed in some Android meeting to always be in sync between (nowadays) files, talk, library and notes. So next time please ask, so that we can check on other apps if this also fits :)

Somehow i had in mind this was overdue for android talk so i didn't check the other apps again. But glad it's okay for the other apps now!
